src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -610,7 +610,7 @@ public override void GenerateViewGeometryFromRenderPackagesAndRequestUpdate(IEnu
             OnRequestViewRefresh();
         }
 
-        protected override void DeleteGeometryForIdentifier(string identifier, bool requestUpdate = true)
+        internal override void DeleteGeometryForIdentifier(string identifier, bool requestUpdate = true)
```

Comment:
See comments above and below about `IWatch3DViewModel`. Following those, this method would become public.